src/pivot-table/hooks/use-column-width.ts

```diff
@@ -112,34 +105,46 @@ export default function useColumnWidth(
     };
 
     let sumOfWidths = 0;
+    const columnWidths = headersData.data[0].map((_, colIdx) => {
+      const colWidth = headersData.data.reduce((accWidth, row, rowIndex, rows) => {
+        const isLastRow = rowIndex === rows.length - 1;
+        const cell = row[colIdx];
+        let width = 0;
+
+        if (cell) {
+          const { id, label, isLocked } = cell;
+          if (id !== PSEUDO_DIMENSION_KEY) {
+            const lockedIconSize = isLocked ? LOCK_ICON_SIZE : 0;
+            width = measureTextForHeader(label) + TOTAL_CELL_PADDING + MENU_ICON_SIZE + lockedIconSize;
+          }
+
+          if (isLastRow) {
+            if (colIdx === headersData.pseudoDimensionColumnIdx) {
+              width = Math.max(
+                width,
+                ...qMeasureInfo.map(({ qFallbackTitle, columnWidth }) => {
+                  const fitToContentWidth = measureTextForDimensionValue(qFallbackTitle) + TOTAL_CELL_PADDING;
```

Comment:
I think this is bugged (on `main` as well), it should be using something like `measureTextForMeasureLabel`. Should try and fix that once this PR is merged.

Reply:
Fixed